Detectors/PHOS/base/include/PHOSBase/Geometry.h

```diff
@@ -0,0 +1,99 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef ALICEO2_PHOS_GEOMETRY_H_
+#define ALICEO2_PHOS_GEOMETRY_H_
+
+//#include <exception>
+#include <string>
+//#include <tuple>
+//#include <vector>
+
+#include <RStringView.h>
+//#include <TGeoMatrix.h>
+#include <TMath.h>
+//#include <TParticle.h>
+//#include <TVector3.h>
+
+//#include "EMCALBase/Constants.h"
+//#include "EMCALBase/GeometryBase.h"
+//#include "MathUtils/Cartesian3D.h"
+
+namespace o2
+{
+namespace PHOS
```

Comment:
please change all namespaces to `phos` (namespaces should be lower case). This will avoid automatic fixing later on (which we will do for the rest of the code).